tools/clang/include/clang/SPIRV/Structure.h

```diff
@@ -303,8 +302,8 @@ EntryPoint::EntryPoint(spv::ExecutionModel em, uint32_t id, std::string name,
     : executionModel(em), targetId(id), targetName(std::move(name)),
       interfaces(interface) {}
 
-DebugName::DebugName(uint32_t id, llvm::Optional<uint32_t> index,
-                     std::string targetName)
+DebugName::DebugName(uint32_t id, std::string targetName,
+                     llvm::Optional<uint32_t> index)
```

Comment:
Also need to sort the initializer for member variables according to their declaration order. Otherwise, GCC/Clang will issue warning.